src/main/java/org/candlepin/service/model/Event.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2009 - 2024 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.service.model;
+
+/**
+ * The Event interface represents a Candlepin event.
```

Comment:
Something that's somewhat bothering me is that we have duplicate class names in 2 packages now (Event/EventAdapter exist in both org.candlepin.audit and org.candlepin.service), and we have nothing that explains their difference. I wouldn't go as far as changing the class names, but at the very least adding something to the class-level javadocs that explains what these specific events are for, but I can't come up with something good enough.

The best I can think of here for example is: `The Event interface represents a Candlepin event for consumption by external components`. 'external' as opposed to Artemis events being used for candlepin-internal purposes such as the job system, and audit log, but the problem is that even Artemis events are sometimes used by external components like Katello, so its not really accurate.

If you can think of something better, you can update it, but if not ignore my comment

Reply:
I agree that is confusing to have both an Event and EventAdapter in two different packages. I updated the class name to `AdapterEvent` to help make the distinction more clear, and I updated the class-level java doc. I couldn't think of a better description than the one you came up with, and so I updated it to that!